fabric-biomes-v1/build.gradle

```diff
@@ -1,2 +1,2 @@
 archivesBaseName = "fabric-biomes-v1"
-version = getSubprojectVersion(project, "1.0.2")
+version = getSubprojectVersion(project, "1.1.0")
```

Comment:
Is it? It removes one public method and otherwise changes Biome arguments to RegistryKey<Biome>?
Granted. that _is_ a breaking change but does that justify changing the module?

Reply:
The entire module is effectively detached from every older version of biome api. Plus these are breaking changes with a module that was never re-released with .2.

The other biomes PR has the consensus by modmuss that naming and packages should be adapted due to an effective full break.

Reply:
Its 100% atleast a 2.0.0. As for a new module no. -v2 would only need to be used where the old API was being kept around for backwards compat.